gpio_controllers/doc/userdoc.rst

```diff
@@ -0,0 +1,55 @@
+.. _gpio_controllers_userdoc:
+
+gpio_controllers
+=====================
+
+This is a collection of controllers for hardware interfaces of type GPIO (``<gpio>`` tag in the URDF).
```

Comment:
```suggestion
A controller for hardware interfaces of type GPIO (``<gpio>`` tag in the URDF) to be able to command the configured GPIOs and as well to publish their state
```

Reply:
I think this was OK: gpio_controllers follow the structure of other collection packages we have (effort_controller), where only a single controller is implemented yet (gpio_controllers/gpio_command_controller, effort_controllers/JointGroupEffortController)

Reply:
Gotcha! 👍🏾